homeassistant/components/pterodactyl/strings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "url": "[%key:common::config_flow::data::url%]",
+          "api_key": "[%key:common::config_flow::data::api_key%]"
+        },
+        "data_description": {
+          "url": "The IP address or hostname of your Pterodactyl server, starting with either 'http://' or 'https://', optionally including the port at the end.",
+          "api_key": "The client (account) API key for accessing your Pterodactyl server."
```

Comment:
why account?

Reply:
From the docs:

> Pterodactyl has two different types of API keys: Client (also known as Account) and Application. Application API keys are not supported, a client API key as described above is required instead.

But I would have no problem with removing "Account" from the description here, if you prefer it?

Reply:
Just checked on my server: It is shown as "Account API key". I'll name it accordingly everywhere 👍